### DISCUSSION TITLE
Logo 票选

### DISCUSSION THREAD

**Discussion by linyinfeng** [NixOS-CN/nixos-cn-artwork#3] [General]
> 本投票将选出 NixOS CN 组织的 logo。
> 截止时间：2023 年 4 月 1 日 0 时（东八区）（`2023-04-01T00:00:00+08:00`）。
> 投票方式：为心仪的 logo upvote（多选）。
> 参加票选：直接上传新的 logo 参加票选。

已截止，最终获选 logo 为 <https://github.com/nixos-cn/nixos-cn-artwork/discussions/3#discussioncomment-5163020>。

提出意见和建议：[Logo 意见建议收集](https://github.com/nixos-cn/nixos-cn-artwork/discussions/2)。

**Comment by linyinfeng** [comment]
@OpenTritium 原设计
![nixos-cn](https://user-images.githubusercontent.com/11229748/222052537-4e88c273-9403-44b8-a5b8-597f9d10a80c.svg)

**Reply by CyrusYip** [reply]
这个比较简洁。

**Comment by linyinfeng** [comment]
变种 1
![nixos-cn-人1](https://user-images.githubusercontent.com/11229748/222052699-1cc30546-1b74-4a92-9f70-964835a53ab1.svg)

**Comment by linyinfeng** [comment]
变种 2
![nixos-cn-人2](https://user-images.githubusercontent.com/11229748/222052747-eb48575a-9ba6-4d71-b252-621a9233414e.svg)

**Reply by OpenTritium** [reply]
这个很棒！

**Reply by QingyaoLin** [reply]
投票+1

**Reply by xc3z** [reply]
好!

**Reply by Aleksanaa** [reply]
确实很不错

**Reply by luochen1990** [reply]
我感觉这个有点花哨了 （笑哭

**Comment by linyinfeng** [comment]
变种 3
![nixos-cn-人3](https://user-images.githubusercontent.com/11229748/222052771-ee0c9e36-edaf-415d-93e5-84e859d95ecc.svg)

**Reply by ClSlaid** [reply]
(人民教育出版社 - 书)x6

**Reply by CyrusYip** [reply]
人民教育出版社是什么梗？

**Reply by OpenTritium** [reply]
> 人民教育出版社是什么梗？

很像以前这个出版社的图标x6

**Reply by CyrusYip** [reply]
我看现在的图标是「双手捧花，外面套个正方形」。有旧图标的链接吗？想看看。

**Reply by xc3z** [reply]
让我想起来宇智波斑的万花筒写轮眼

**Comment by linyinfeng** [comment]
变种 4
![nixos-cn-人4](https://user-images.githubusercontent.com/11229748/222052802-c5a0dee4-c7b2-4ff6-a936-0ee53e05c92d.svg)

**Reply by OpenTritium** [reply]
~nbuntu~

**Comment by YisuiDenghua** [comment]
![nixoscn_fixed](https://user-images.githubusercontent.com/102890144/222110784-b5c4435f-9a60-49d0-a85c-b4083b28ecd0.png)

**Reply by vicerlin** [reply]
这个好憨

**Comment by fjh1997** [comment]
![00015-424480840](https://user-images.githubusercontent.com/25281816/222120268-38a26870-9057-4c1c-ae03-cb788de910d2.png)

**Reply by OpenTritium** [reply]
中间那个图形是什么(ﾟoﾟ;

**Comment by MidAutumnMoon** [comment]
看，就连它的尾巴也很开心。

![final2](https://user-images.githubusercontent.com/27620522/222125252-0b1574bc-fc2d-46a6-a87c-37c1d943201f.svg)

**Comment by hehongbo** [comment]
![Untitled-2-01](https://user-images.githubusercontent.com/665472/222174063-34c4e95d-76fa-459f-804a-b6827d8068f3.png)

**Comment by Thaumy** [comment]
灵感来自榫卯结构）
![nix](https://user-images.githubusercontent.com/28834579/225358986-e9b807a6-0c8c-481b-bbe5-6d2c7a89e0ab.svg)

**Reply by OpenTritium** [reply]
很好的设计，爱来自中国

**Reply by LostAttractor** [reply]
这个好棒，几何的美感

**Reply by CHN-beta** [reply]
这个有种纪念碑谷的感觉

**Reply by vicerlin** [reply]
有点冬奥的感觉了

**Comment by YisuiDenghua** [comment]
![kor](https://github.com/NixOS-CN/nixos-cn-artwork/assets/102890144/54cfbfca-0b97-4f0c-ba0d-3ea94db7664c)
图中文字是朝鲜语谚文，分别是 n, i, k, s
